### PR TITLE
Addition of Portuguese language

### PR DESCRIPTION
**Pull request by MishoKanaria**
This addition was contributed by @DOMSX1

### CODE REVIEW

File: .vitepress/config.mts [hunk 1]
```diff
@@ -845,6 +845,377 @@ export default withMermaid(
...
+                    { text: "Ver estatísticas", link: "pt/statistics" },
+                    { text: "Explorar blocos", link: "pt/explore-blocks" },
+                  ],
+                },
+                { text: "Transferir ativos", link: "pt/transfer" },
+                { text: "Verifique o saldo", link: "pt/check-balance" },
+                { text: "Registrar um ativo", link: "pt/register-an-asset" },
+                { text: "Rampa de acesso (on-ramp)", link: "pt/on-ramp" },
+                {
+                  text: "Staking (depósito remunerado, uma premiação em moedas digitais emitida como recompensa por ajudar na segurança da rede)",
```
Comment (AndreyKortosov) [inline .vitepress/config.mts]: I'm not sure we should explain in the menu what's Staking, it's better to see only 1 word

File: .vitepress/config.mts [hunk 2]
```diff
@@ -845,6 +845,377 @@ export default withMermaid(
...
+                        {
+                          text: "Estabelecendo canal HRMP",
+                          link: "pt/establishing-hrmp-channel",
+                        },
+                      ],
+                    },
+                  ],
+                },
+                { text: "Sistema de referência", link: "pt/referral" },
+                { text: "NFT ou token não fungível (ativo digital único)", link: "pt/nft" },
```
Comment (AndreyKortosov) [inline .vitepress/config.mts]: Here it's better to say 'NFT (ativo digital único)'

File: .vitepress/config.mts [hunk 3]
```diff
@@ -845,6 +845,377 @@ export default withMermaid(
...
+                              link: "pt/ceres/token-locker",
+                            },
+                            { text: "Governança", link: "pt/ceres/governance" },
+                            { text: "Plataforma de lançamento", link: "pt/ceres/launchpad" },
+                          ],
+                        },
+                        {
+                          text: "Demeter",
+                          items: [
+                            { text: "Farming ou cultivo de rendimento (depósito de ativos para empréstimos em troca de remuneração)", link: "pt/ceres/demeter-farming" },
```
Comment (AndreyKortosov) [inline .vitepress/config.mts]: I think will be enough to see Farming and inside the page they will find explanation

**Review by AndreyKortosov — COMMENTED** [review]
I left some comments for titles in the menu (comfig.mts)

**Review by AndreyKortosov — APPROVED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**